MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
 
 The data grid supports to display the concise information about the bound data objects using summaries. The control provides the following summary types:
 
+* **Caption Summary** - Used to display the summary information in the caption of the group.
+* **Group Summary** - Used to display summary information of data objects in each group.
 * **Table Summary** - Used to display the summary information at top and/or bottom in SfDataGrid.
 
 
-![DataGrid with summary rows](Images\Table-Summary\ShowSummaryInRow.png) 
+![DataGrid with summary rows](Images\Caption-Summary\Summaries.png) 
 
 Summary rows are represented by using the [DataGridSummaryRow](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSummaryRow.html) that hold summary information of columns in the [SummaryColumns](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSummaryRow.html#Syncfusion_Maui_DataGrid_DataGridSummaryRow_SummaryColumns) property . The `SummaryColumns` contains the collection of [DataGridSummaryColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridSummaryColumn.html) which carries name, format, and summary aggregate type of the column.
 
 Derive additional information from the data like sum, average, maximum, minimum, and count using summaries in the data grid. These summary values can be computed for groups or for the entire control using `DataGridSummaryRow` and `DataGridSummaryColumn` that implements [ISummaryRow](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ISummaryRow.html) and [ISummaryColumn](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.ISummaryColumn.html) interfaces.
 
 N> The Summary does not refresh with data. To update the summary for the newly added row, or for the modified summary column, set the [SfDataGrid.View.LiveDataUpdateMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Data.LiveDataUpdateMode.html) to `LiveDataUpdateMode.AllowDataShaping` or `LiveDataUpdateMode.AllowSummaryUpdate`.
 
+## Caption summaries
+
+The data grid provides built-in support for caption summaries. The caption summary value calculated based on the records in a group. The summary information will be displayed in the caption of group.
+
+### Formatting built-in caption summary
+
+By default, caption summary rows are displayed with the [SfDataGrid.GroupCaptionTextFormat]() property.
```

Comment:
By default, the caption summary rows in the data grid are displayed using the [SfDataGrid.GroupCaptionTextFormat](https://chat.openai.com/) property.

The default group caption format is {ColumnName}: {Key} - {ItemsCount} Items.

ColumnName: This displays the name of the currently grouped column.
Key: This displays the key value of the group.
ItemsCount: This displays the number of items in a group.
DataGrid with formatting in caption summary row

You can customize the group caption text format by setting the SfDataGrid.GroupCaptionTextFormat property. The code example below illustrates how to customize the group caption text in the data grid:

Reply:
Modified.